packages/gp2-components/src/templates/OnboardWelcome.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  Caption,
+  Divider,
+  Headline2,
+  Subtitle,
+  Paragraph,
+  Link,
+  pixels,
+  Button,
+  informationInfo500,
+} from '@asap-hub/react-components';
+import { logout } from '@asap-hub/routing';
+import { css } from '@emotion/react';
+import CardWithCornerBackground from '../molecules/CardWithCornerBackground';
+import BasicLayout from './BasicLayout';
+
+const { mobileScreen } = pixels;
+
+const containerStyles = css({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'left',
+});
+
+const footerStyles = css({
+  justifyContent: 'space-between',
+  [`@media (min-width: ${mobileScreen.max}px)`]: {
+    display: 'flex',
+    flexDirection: 'row-reverse',
+    button: {
+      maxWidth: 'fit-content',
+    },
+    a: {
+      maxWidth: 'fit-content',
+    },
+  },
+});
+
+const OnboardWelcome: React.FC<Record<string, never>> = () => (
+  <BasicLayout>
+    <div css={containerStyles}>
+      <header>
+        <Headline2 noMargin>Welcome to the GP2 Hub</Headline2>
```

Comment:
noMargin hasMargin 👎

Reply:
Please try to call out people adding props like this. Should just be margin={false} to disable margin. Its what we have used consistently up to this point.